### PR TITLE
🔥 Sezonluk Dizi and TRanimeizle Presences - Fixes for Sinefy Presence

### LINKED ISSUE #2273: Tranimeizle | www.tranimeizle.net
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username? kira#9507

**What's the name of the service? www.tranimeizle.net | Tranimeizle

**What should the Presence display Tranimeizle is a site where you can watch Turkish subtitle anime & read manga.

**If possible, please provide a logo for the service (512x512 minimum) 
Untitled-2

### PR DESCRIPTION
**Pull request by eggsy**
### Sezonluk Dizi
Another Turkish video streaming and TV series website.

moment0

moment1

moment2

moment3

moment4

moment5

### TRanimeizle Presence
A Turkish anime listing and watching platform. This PR closes #2273 made by the website author @tranimeizle (I believe).

moment1

moment2

moment3

moment4

### Sinefy Iframe Fix
It wasn't displaying timestamps. It does now.

moment1

moment2

### CODE REVIEW

**Review by Bas950 — APPROVED** [review]

**Comment by eggsy** [comment]
Sorry Bas 🥺

File: websites/S/Sezonluk Dizi/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
```
Comment (Bas950) [inline websites/S/Sezonluk Dizi/dist/metadata.json]: please use latest schema (v1.3)

File: websites/S/Sezonluk Dizi/presence.ts [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+      timestamps = sezonlukDizi.getTimestamps(
+        video.currentTime,
+        video.duration
+      );
+
+    presenceData.details = showTitle;
+    presenceData.state = episode;
+
+    // Set timestamps
+    presenceData.startTimestamp = timestamps[0];
```
Comment (Bas950) [inline websites/S/Sezonluk Dizi/presence.ts:123]: remove comment

File: websites/S/Sezonluk Dizi/presence.ts [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+    presenceData.buttons = [
+      {
+        label: "Bölümü İzle",
+        url: location.href
+      }
+    ];
+
+    // Set playing/paused text
+    presenceData.smallImageKey = video.paused ? "pause" : "play";
```
Comment (Bas950) [inline websites/S/Sezonluk Dizi/presence.ts:139]: same here

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/S/Sezonluk Dizi/dist/metadata.json (hunk 1 above)
Comment (eggsy) [reply websites/S/Sezonluk Dizi/dist/metadata.json]: This is an old commit I guess. It's already set to 1.3

**Review by StrikerFRFX — APPROVED** [review]
Eggsy I hope this works and I don't need the second glance I should take xd

**Review by Bas950 — APPROVED** [review]

**Pull request merged**